ui/desktop/app/styles/app.scss

```diff
@@ -530,3 +530,111 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     background-color: var(--token-color-palette-neutral-50);
   }
 }
+
+.details-screen-body {
+  .rose-layout-body-content {
+    margin-top: 1.5rem;
+  }
+}
+
+// Session Details - Credentials Panel
+.credential-panel-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 0.5rem;
+
+  .title {
+    font-weight: var(--token-typography-font-weight-bold);
+  }
+
+  /* stylelint-disable selector-class-pattern */
+  .hds-form-field--layout-flag {
+    margin-bottom: 0;
+  }
+}
+
+.credential-panel-body {
+  border: 0.063rem solid var(--token-color-border-primary);
+  border-radius: 0.313rem;
```

Comment:
These are incredibly specific rems, should these be pixels?

Reply:
I am fine with using pixels. My understanding is we are using rem as our standard. Apologies if that changed and I missed it.

Reply:
I think that's true for typography and layout but I don't think that makes sense for something like `border-radius`, would it make sense for `border-radius` to change depending on the root font size?

Reply:
Good point. I will update this.